index.html

```diff
@@ -114,7 +114,15 @@ <h1>Glossary</h1>
       that handled the chain of requests initiated by this button
       being pressed.
     </dd>
-  </dl>
+    </dl>
+    <dl>
+    <dt><dfn data-lt="opaque">Opaque value</dfn></dt>
+    <dd>
+      An opaque value refers to a value that can only be understood,
```

Comment:
Incorrect comma use.

Per [purdue online writing lab notes on comma use](): 

>13. Don't put a comma between the two verbs or verb phrases in a compound predicate.
>
> INCORRECT: INCORRECT: We laid out our music and snacks, and began to study.
> CORRECT: INCORRECT: We laid out our music and snacks and began to study.

```suggestion
      An opaque value refers to a value that can only be understood
```